Rebuild-Black-Business/RBB-Website
# Add Page Hero

# Describe your PR
This PR adds a component called `<PageHero />` which is used to display a title, subtitle, and header image at the top of a given page, based on the headers from [these](https://www.figma.com/file/Lo1ZcBOO9t4QRbX2I0PwBQ/RBB?node-id=933%3A6032) [mockups](https://www.figma.com/file/Lo1ZcBOO9t4QRbX2I0PwBQ/RBB?node-id=0%3A1)

Related to #n/a
Fixes #n/a

## Pages/Interfaces that will change
`/businesses`, `/allies`, possibly more

### Screenshots / video of changes

This is a WIP feature, I'll update pictures in conversation below as I continue work.  For now this is where I'm at:

![image](https://user-images.githubusercontent.com/1844496/84070828-bd0c9e80-a99a-11ea-8fcb-06973031e8a2.png)


## Steps to test

1. open the deploy preview
1. click `Businesses` and check it out at full size and responsive size

### Additional notes

Some other things I think we might need to tackle here:
- [x] a Heading component - this style will be shared amongst a bunch of pages, and in the spirit of DRY, should be split out into its own component
- [x] a subtitle component, for the same reasons
